src/data/faq_de.json

```
@@ -439,6 +439,15 @@
+                        "active": true,
+                        "textblock": [
+                            "<div class='container'><div class='row'><div class='col-sm'><a href='https://www.coronawarn.app/assets/screenshots/2-14/de/android/PersonDetailsFragment_cwa_2.png' target='_blank' rel='noopener'><img src='../../assets/screenshots/2-14/de/android/PersonDetailsFragment_cwa_2.png' alt='image' width='300'></a></div><div class='col-sm'><p>Die Auffrischimpfung (Boosterimpfung) ist ein Impfvorgang, der nach der Grundimmunisierung den nachlassenden Impfschutz auffrischen soll. In den meisten Fällen, aber nicht immer, findet Sie gemäß den Empfehlungen der Behörden nach einer Zeit von etwa 5-6 Monaten nach dem letzten Impfvorgang statt. Typisches Beispiel: Erfolgte der Abschluss der Impfreihe zur Grundimmunisierung am 08.07.2021, dann könnte eine Auffrischimpfung am 08.01.2022 erfolgen.</p><p>Jede Auffrischimpfung kann durch ein digitales COVID-Zertifikat dokumentiert werden, das Sie in die Corona-Warn-App einlesen können. <b>*Wichtig*</b>: Bitte achten Sie schon direkt beim Ausstellen des Zertifikats darauf, dass die persönlichen Daten (Titel, Name, Vornamen, und Geburtsdatum) passend zu den bereits existierenden Impfzertifikaten ausgewiesen werden. Nur so kann die Corona-Warn-App das neue Zertifikat der Auffrischimpfung der Grundimmunisierung zuordnen (Beispiel: gleicher akademischer Titel oder Angabe des vollständigen Doppelnamens). Alle Zertifikate einer Person sollten in der Corona-Warn-App gruppiert angezeigt werden, siehe Abbildung.</p><p>Auffrischimpfungen bieten einen sofortigen Impfschutz.</p><p>Basierend auf den bisherigen Impfvorgängen, wird das erste Zertifikat einer Auffrischimpfung mit '3 von 3' (bei vorheriger Grundimmunisierung mit 2 Dosen) oder mit '2 von 2' (als Impfoptimierung nach dem Wirkstoff Janssen von Johnson & Johnson) ausgestellt. Im letzteren Fall können die CovPassCheck-App, Corona-Warn-App und CovPass-App dies aber aus technischen Gründen nicht in allen Fällen korrekt anzeigen. Siehe FAQ-Artikel: <a href='https://www.coronawarn.app/de/faq/#vac_booster_jj' target='blank' rel='noopener noreferrer'>Warum erscheint in der Corona-Warn-App eine 14-tägige Wartezeit bis zum vollen Impfschutz nach einer Auffrischungsimpfung?</a></p><p>Die Kennzeichnung '1 von 1' im Zertifikat einer Auffrischimpfung zeigt an, dass es sich um den ersten und einzigen Impfvorgang nach einer Genesung handelt. Eine Ausnahme bilden Impfungen mit Johnson & Johnson. Siehe FAQ-Artikel: <a href='https://www.coronawarn.app/de/faq/#vac_booster_jj' target='blank' rel='noopener noreferrer'>Warum erscheint in der Corona-Warn-App eine 14-tägige Wartezeit bis zum vollen Impfschutz nach einer Auffrischungsimpfung?</a></p></div></div></div>"
+                        ]
+                    },
+
                     {
                         "title": "Warum erhalte ich eine Benachrichtigung über eine Impfempfehlung der Ständigen Impfkomission (STIKO)?",

```

```suggestion
                        "title": "Warum erhalte ich eine Benachrichtigung über eine Impfempfehlung der Ständigen Impfkommission (STIKO)?",
```
See https://www.rki.de/DE/Content/Kommissionen/STIKO/stiko_node.html as reference for correct spelling.

---

Also needs changing in line 347.